src/lib/stitching/causality/stitching.ts

```diff
@@ -22,9 +28,57 @@ export const causalityStitchingEnvironment = ({
       extend type AuctionsLotStanding {
         saleArtwork: SaleArtwork
       }
+
+      extend type AuctionsLotState {
+        floorSellingPrice2: Money
+      }
     `,
 
     resolvers: {
+      AuctionsLotState: {
+        floorSellingPrice2: {
+          fragment: gql`
+            ... on AuctionsLotState {
+              internalID
+              floorSellingPrice
+            }
+          `,
+          type: Money,
+          resolve: async (
+            { internalID, floorSellingPrice },
+            _args,
+            context,
+            info
+          ) => {
+            const { currency } = await context.saleArtworkRootLoader(internalID)
+            const { units: minor } = floorSellingPrice
+            const major = await moneyMajorResolver(
+              { cents: minor, currency },
+              {},
+              context,
+              info
+            )
+
+            // return { cents: minor, currency }
+
+            return {
+              currencyCode: currency,
+              minor,
+              major,
+              display: formatMoney(major, symbolFromCurrencyCode(currency)),
+            }
```

Comment:
One option is to resolve all fields manually. With some [extraction](https://github.com/artsy/metaphysics/pull/2805/files#diff-c7c0cff0902f5e31e018c4c77fe43c6c9647323cb8621d5e96ec57222491ab8cR168), this doesn't feel _too_ bad, but still pretty manual.